### ISSUE TITLE
Running Django

### ISSUE THREAD

**Issue by ifuchs** [holzschu/Carnets#158]
I installed Django using !pip but how can I run Django commands to develop and test an app?

**Comment by holzschu** [comment]
We're talking about advanced use of Carnets here. If the commands have been installed by the pip command, you can launch them using `!command `. There was a similar issue in a-Shell: https://github.com/holzschu/a-shell/issues/72

**Comment by ifuchs** [comment]
Ok, so after installing django without error. I ran !django-admin startproject DemoApp and a folder DemoApp was created with the expected files. Now, how do I run manage.py runserver?
I tried to cd into the DemoApp folder but !ls doesn't return the contents of the folder. Where did I go wrong?

**Comment by holzschu** [comment]
So, I have never run Django, but I would expect something like: `!python manage.py runserver --noreload` to work.

**Comment by ifuchs** [comment]
Yes, that makes sense thoughit can't find that file to execute. I tried doing !cd DemoApp before running the command but the directory (based on !pwd) is not changing.

**Comment by holzschu** [comment]
Good point. How about `!python DemoApp/manage.py runserver --noreload`, or creating a notebook inside the DemoApp folder, then calling `!python manage.py runserver --noreload` from that notebook?

For running Django specifically, maybe `a-Shell` would be a more appropriate app.

**Comment by ifuchs** [comment]
Probaby but I wanted to see if it would run in Carnets and now it does! (at least as far as being able to get to the Django local webserver). Thanks!
